### PR TITLE
Add Softsign fused op

### PR DESCRIPTION
**Pull request by Ewa21**

### CODE REVIEW

File: src/ngraph/op/fused/softsign.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const auto one_node = op::Constant::create(data.get_element_type(), data_shape, {1});
+
+    return {data / (std::make_shared<op::Abs>(data) + one_node)};
+}
+
+std::shared_ptr<Node> op::Softsign::copy_with_new_args(const NodeVector& new_args) const
+{
+    check_new_args_count(this, new_args);
+    return std::make_shared<Softsign>(new_args.at(0));
+}
```
Comment (tsocha) [inline src/ngraph/op/fused/softsign.cpp]:
```suggestion
}

```

File: src/ngraph/op/fused/softsign.hpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            /// \param   data    Input tensor.
+            ///
+            Softsign(const Output<Node>& data);
+
+            virtual NodeVector decompose_op() const override;
+            virtual std::shared_ptr<Node>
+                copy_with_new_args(const NodeVector& new_args) const override;
+        };
+    }
+}
```
Comment (tsocha) [inline src/ngraph/op/fused/softsign.hpp]:
```suggestion
}

```

File: src/ngraph/op/fused/softsign.cpp [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::Softsign::decompose_op() const
+{
+    const auto data = input_value(0);
+    const auto data_shape = data.get_shape();
+    const auto one_node = op::Constant::create(data.get_element_type(), data_shape, {1});
+
+    return {data / (std::make_shared<op::Abs>(data) + one_node)};
```
Comment (GlebKazantaev) [inline src/ngraph/op/fused/softsign.cpp]: As I know `/` and `+` operators are overloaded for v0 ops. Let's use v1 ops here.

**Comment by postrational** [comment]
This operation was removed from `v1`.

**Pull request closed without merging**